python/paddle/fluid/layers/nn.py

```diff
@@ -7580,3 +7582,97 @@ def hash(input, hash_size, num_hash=1, name=None):
         attrs={'num_hash': num_hash,
                'mod_by': hash_size})
     return out
+
+
+def log_loss(input, label, epsilon=1e-4, name=None):
+    """
+    **Negative Log Loss Layer**
+
+    This layer accepts input predictions and target label and returns the
+    negative log loss.
+
+    .. math::
+
+        Out = -label * log(input + epsilon)
```

Comment:
can use latex format like `\log{(input + \epsilon)}` for better document view. try using https://www.codecogs.com/latex/eqneditor.php